README.md

```diff
@@ -13,30 +13,135 @@ Bestiary is a tool to manage this kind of description using a web based interfac
 
 It also provides an interface to connect with [GrimoireLab](http://grimoirelab.github.io) to work as analytics scope manager.
 
-# How to run it?
 
-Clone the repository, and we recommend to set up a Python virtual environment to run it
+## Requirements
 
+- Python >= 3.6
+- Poetry >= 1.1.0
+- MySQL >= 5.7 or MariaDB >= 10.2
+- Django = 3.1
+- Graphene-Django >= 2.0
+- Django-graphql-jwt
+
+You will also need some other libraries for running the tool, you can find the whole list of dependencies in [pyproject.toml](pyproject.toml) file.
+
+## Installation
+
+### Getting the source code
+
+Clone the repository, and change to the `unicorn` branch
+
+```
+$ git clone https://github.com/chaoss/grimoirelab-bestiary
+$ git checkout unicorn
+```
+
+### Backend
+
+#### Prerequisites
+
+##### Poetry
+
+We use [Poetry](https://python-poetry.org/docs/) for managing the project.
+You can install it following [these steps](https://python-poetry.org/docs/#installation).
+
+##### mysql_config
+
+Before you install Bestiary tool you might need to install `mysql_config`
+command. If you are using a Debian based distribution, this command can be
+found either in `libmysqlclient-dev` or `libmariadbclient-dev` packages
+(depending on if you are using MySQL or MariaDB database server). You can
+install these packages in your system with the next commands:
+
+* **MySQL**
+
+```
+$ apt install libmysqlclient-dev
+```
+
+* **MariaDB**
+
+```
+$ apt install libmariadbclient-dev
+```
+
+#### Installation and configuration
+
+Install the required dependencies (this will also create a virtual environment).
+```
+$ poetry install
+```
+
+Activate the virtual environment:
+```
+$ poetry shell
+```
+
+Migrations, and create a superuser:
 ```
-$ python3 -m venv bestiary
-$ source bestiary/bin/activate
+(.venv)$ ./manage.py makemigrations --settings=config.settings.devel
```

Comment:
This line shouldn't be needed. It's only needed when developing and the model has changed.